dao/application_dao.go

```diff
@@ -79,3 +80,57 @@ func (a *ApplicationDaoImpl) Delete(id *int64) error {
 func (a *ApplicationDaoImpl) Tenant() *int64 {
 	return a.TenantID
 }
+
+func (a *ApplicationDaoImpl) BulkMessage(id *int64) (map[string]interface{}, error) {
+	application := &m.Application{ID: *id}
+	resource := DB.Preload("Source.Tenant").Preload("Source.Applications.Tenant").Preload("Source.Endpoints.Tenant").Find(&application)
```

Comment:
Super small suggestion just for readability + styling of these really long method chains:

```suggestion
	resource := DB.
		Preload("Source.Tenant").
		Preload("Source.Applications.Tenant").
		Preload("Source.Endpoints.Tenant").
		Find(&application)
```

Can we apply this to all implementations of this method?

Reply:
It is now on one place: https://github.com/RedHatInsights/sources-api-go/pull/49/commits/00c6a5c39881ec0b0bbc8d8d8c1f692f5513e1e0